roles/kubernetes-apps/ansible/templates/coredns-svc.yml.j2

```diff
@@ -20,9 +20,11 @@ spec:
     - name: dns
       port: 53
       protocol: UDP
+      targetPort: "dns"
```

Comment:
HI @liuxu623 

The `targetPort` is should be setted as int not string.
Does the svc.yml needs to changed ? :-)

Reply:
`targetPort` support set to port name, like
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: nginx
  labels:
    app.kubernetes.io/name: proxy
spec:
  containers:
  - name: nginx
    image: nginx:stable
    ports:
      - containerPort: 80
        name: http-web-svc

---
apiVersion: v1
kind: Service
metadata:
  name: nginx-service
spec:
  selector:
    app.kubernetes.io/name: proxy
  ports:
  - name: name-of-service-port
    protocol: TCP
    port: 80
    targetPort: http-web-svc
```
https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports

Reply:
svc.yml needs to changed, because if we change the default CoreDNS port, like 9053, `targetPort` also need changed.

Reply:
Thanks @liuxu623

/lgtm